pkg/legacy/converter.go

```diff
@@ -155,8 +157,16 @@ func buildProxySettings(agentConfig Config) (string, error) {
 			u.User = url.User(user)
 		}
 	}
+	// check if url starts with http/https
+
+	if strings.HasPrefix(u.String(), "https") {
```

Comment:
What I told you was wrong, the proxy in agent5 was applied to both `http` and `https`. So we should set both.